eng/versioning/version_client.txt

```diff
@@ -36,3 +36,4 @@ com.azure:azure-storage-queue;12.2.0-beta.1;12.2.0
 # Format;
 # unreleased_<groupId>:<artifactId>;dependency-version
 unreleased_com.azure:azure-core-amqp;1.0.1
+unreleased_com.azure:azure-core;1.3.0-beta.1
```

Comment:
I am not sure why we are using unreleased for this. There's also this on L8 - `com.azure:azure-core;1.2.0;1.3.0-beta.1`

Reply:
Because I think with recent updates we wanted all libraries to use the released version of core and not the dependency version. And if we wanted to use the current version we will have to include it this way.